### PR TITLE
Note flakey faculty data

### PR DESCRIPTION
**Pull request by jackharrhy**
also note that since the script was ran, `~mungolf` is no more? :thinking:

### CODE REVIEW

File: content/resources/faculty/contents.lr [hunk 1]
```diff
@@ -6,49 +6,50 @@ body:
 
 Below is a _noncomprehensive_ list of homepages for various computer science faculty members, potentially outdated/dead links.
 
-* Adrian Fiech - [afiech](http://www.cs.mun.ca/~afiech/)
-* Andrew Vardy - [av](http://www.cs.mun.ca/~av/)
+Note: some users in this list might be duplicated / not really 'people', it's simply based on ever user within the faculty directory using [this](https://github.com/MUNComputerScienceSociety/Website/blob/master/content/resources/faculty/generate-faculty-page.fish) script.
```
Comment (nint8835) [inline content/resources/faculty/contents.lr]: Typo: "user" should be "users"

File: content/resources/faculty/generate-faculty-page.fish [hunk 2]
```diff
@@ -30,6 +30,8 @@ function faculty_page
   echo ""
   echo "Below is a _noncomprehensive_ list of homepages for various computer science faculty members, potentially outdated/dead links."
   echo ""
+  echo "Note: some users in this list might be duplicated / not really 'people', it's simply based on ever user within the faculty directory using [this](https://github.com/MUNComputerScienceSociety/Website/blob/master/content/resources/faculty/generate-faculty-page.fish) script."
```
Comment (nint8835) [inline content/resources/faculty/generate-faculty-page.fish]: Typo: "user" should be "users"

Comment (jackharrhy) [reply content/resources/faculty/generate-faculty-page.fish]: d706aca627f0fa337010b297e271fedf8eba2e91

File: content/resources/faculty/contents.lr (hunk 1 above)
Comment (jackharrhy) [reply content/resources/faculty/contents.lr]: 702d80f0736974fb003d1c79a730bcde820b3cf5

**Pull request merged**